domain/service/service_inf/invitation_service_inf.go

```diff
@@ -6,9 +6,13 @@ import (
 )
 
 type IInvitationService interface {
-	CreateGroupInvitation(request GroupInputDTO, groupID UUID) error
+	CreateGroupInvitation(request GroupInvitationDTO) error
 
 	AcceptGroupInvitation(invitation UUID, userID UUID) error
```

Comment:
Given that we now have dedicated services and handlers for invitations, it seems reasonable to merge AcceptGroupInvitation and DeclineGroupInvitation back into a single function called handleGroupInvitation. They share some logic, and reintroducing the "accept" boolean in GroupInvitationInputDTO would allow us to handle both acceptance and rejection within the same function. Now when we have a dedicated handler/service it's more clear what such endpoint does. What do you think?

Reply:
For now, i can not tell you whether this is a good or bad idea. You are right that we can save some lines of code but i would wait till we know from the frontend which way to handle invitations is the preferred one. I will create an issue for that so that we keep it in mind.

Reply:
From the frontend‘s perspective, it doesn't matter much. We can either go with one endpoint and an extra JSON parameter or use two endpoints without a parameter. In terms of overhead involved, it's pretty much the same. 

An extra issue for that is fine, we can also keep it that way 👍